Source/WebGPU/WebGPU/Device.h

```diff
@@ -126,7 +126,7 @@ class Device : public WGPUDeviceImpl, public ThreadSafeRefCountedAndCanMakeThrea
 
     bool isValid() const { return m_device; }
     bool isLost() const { return m_isLost; }
-    const WGPULimits& limits() const { return m_capabilities.limits; }
+    const WGPULimits& limits() const SWIFT_RETURNS_INDEPENDENT_VALUE { return m_capabilities.limits; }
```

Comment:
But this value isn't independent, right? It's a reference to memory inside this object.

Reply:
m_capabilities is a data member of Device, and &m_capabilities.limits is an address inside Device, no?

I’m not sure why it helps that CommandEncoder outlives Device. If CommandEncoder calls this function, and then outlives Device, that’s what creates a risk of use after free. 

In any case, we want to use local reasoning for lifetimes, and locally this annotation is incorrect. The returned value is a pointer into the Device object, so it is not independent.